### DISCUSSION TITLE
memory_limit

### DISCUSSION THREAD

**Discussion by vico-gs** [spatie/image#195] [Q&A]
Hello !
I implemented spatie in my project and started to check the memory usage that it takes to upload an image, optimize it and save it.
I realized that for an image of 9.5 Mb it uses 385 Mb. I understand that the treatment of an image is heavy, but is that normal ? It sounds a lot to me.
Do you have the same return from your spatie usage or does that come from my code ?

**Comment by kropacek-ogsoft** [comment]
Hello, I have the same problem. I had to increase php memory limit to 1024M. Does anybody have the same problem?
